src/assets/src/components/manage.tsx

```diff
@@ -26,6 +30,9 @@ function ManageQueueTable(props: ManageQueueTableProps) {
                 <Link to="/add_queue" tabIndex={-1}>
                     <Button variant='success' aria-label='Add Queue'>+ Add Queue</Button>
                 </Link>
+                {props.queues.length && props.onAllQueueHistoryDownload &&(
```

Comment:
Number '0' was shown next to the "+ Add Queue" button with zero queue item.

![image](https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/359944/8350f176-e25c-4601-ac0f-5fad45c83c50)

Reply:
I noticed this too. Addressed in the recent commit, the button should be available to press even if there's no queues in the UI